### ISSUE TITLE
[Paid Feature Request] Persistant Vehicle/Mount 

### ISSUE THREAD

**Issue by xnitro1** [suriyun-production/mmorpg-kit-docs#1924]
Hey there, i know you wont add any new feature requests for free anymore. Therefore my question:

I want the vehicles to be persistant so they wont be reseted everytime the server restarts. How much would that cost and how fast could it be made?

best regards

**Comment by xnitro1** [comment]
yes, like in all other survival games too. When a players enters the car and drive vom place A to B then the car should be there  at place B where he left it after a restart (i believe this needs the car to be added in the database with position and rotation)

**Comment by xnitro1** [comment]
any news if its possible? :)

**Comment by insthync** [comment]
It is possible, but I want to focus on the console platforms UI feature for now.

**Comment by xnitro1** [comment]
ok thx for the information
